examples/gno.land/p/demo/auction/auction.gno

```diff
@@ -0,0 +1,94 @@
+package auction
+
+import (
+	"std"
+	"strconv"
+	"time"
+
+	"gno.land/p/demo/ownable"
+	"gno.land/p/demo/ufmt"
+)
+
+// Auction struct
+type Auction struct {
+	Owner       ownable.Ownable // Embeds Ownable to handle ownership
+	Title       string
+	Description string
+	Begin       time.Time
+	End         time.Time
+	StartingBid std.Coin
+	Bids        []*Bid
+	State       string // "upcoming", "ongoing", or "closed"
+}
+
+// Bid struct
+type Bid struct {
+	Bidder std.Address
+	Amount std.Coin
+}
+
+var (
+	auctionList []*Auction
+	currentTime time.Time
+)
```

Comment:
These don't make sense in the package / are not used in the test

Reply:
You're right. it's removed [61066d74](https://github.com/gnolang/gno/pull/2265/commits/61066d74bc8e420572010a7d9c6056f1e2a7a099)